core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -507,6 +507,28 @@ class RealityTreeReference extends RealityModelTileTree.Reference {
   }
 
   public get classifiers(): SpatialClassifiers | undefined { return undefined !== this._classifier ? this._classifier.classifiers : undefined; }
+  public createDrawArgs(context: SceneContext): TileDrawArgs | undefined {
+    // For global reality models (OSM Building layer only) - offset the reality model by the BIM elevation bias.  This would not be necessary
+    // if iModels had their elevation set correctly but unfortunately many GCS erroneously report Sea (Geoid) elevation rather than
+    // Geodetic.
+    const tree = this.treeOwner.load();
+    if (undefined === tree)
+      return undefined;
+
+    if (!this._iModel.isGeoLocated ||
+      this.computeWorldContentRange().diagonal().magnitude() < 2 * Constant.earthRadiusWGS84.equator)

```

Comment:
Is this something you can compute once and store?

Reply:
The inverse of this is computed once in `ContextRealityModelState.isGlobal`.
`RealityModelTileTree.isContentUnbounded` computes it each time; so does `RealityModelTileTree.unionFitRange`.
It'd be good to compute+store it in just one place.

Reply:
Yes... Done.